proposals/numeric-intptr.md

```diff
@@ -0,0 +1,344 @@
+# Numeric IntPtr
+
+## Summary
+[summary]: #summary
+
+This is a revision on the initial native integers feature ([spec](https://github.com/dotnet/csharplang/blob/main/proposals/csharp-9.0/native-integers.md)), where the `nint`/`nuint` types were distinct from the underlying types `System.IntPtr`/`System.UIntPtr`.
+In short, we now treat `nint`/`nuint` as simple types aliasing `System.IntPtr`/`System.UIntPtr`, like we do for `int` in relation to `System.Int32`.
+
+## Design
+[design]: #design
+
+### 8.3.5 Simple types
+
+C# provides a set of predefined `struct` types called the simple types. The simple types are identified through keywords, but these keywords are simply aliases for predefined `struct` types in the `System` namespace, as described in the table below.
+
+**Keyword** | **Aliased type**
+----------- | ------------------
+  `sbyte`   |   `System.SByte`
+  `byte`    |   `System.Byte`
+  `short`   |   `System.Int16`
+  `ushort`  |   `System.UInt16`
+  `int`     |   `System.Int32`
+  `uint`    |   `System.UInt32`
+  **`nint`**     |   **`System.IntPtr`**
+  **`nuint`**    |   **`System.UIntPtr`**
+  `long`    |   `System.Int64`
+  `ulong`   |   `System.UInt64`
+  `char`    |   `System.Char`
+  `float`   |   `System.Single`
+  `double`  |   `System.Double`
+  `bool`    |   `System.Boolean`
+  `decimal` |   `System.Decimal`
+
+\[...]
+
+### 8.3.6 Integral types
+
+C# supports **eleven** integral types: `sbyte`, `byte`, `short`, `ushort`, `int`, `uint`, **`nint`, `nuint`**, `long`, `ulong`, and `char`. \[...]
+
+## 8.8 Unmanaged types
+
+In other words, an __unmanaged_type__ is one of the following:
```

Comment:
>__unmanaged_type__ [](http://example.com/codeflow?start=19&length=18)

Consider using italics rather than bold to match the existing spec and to avoid indicating a change.